domains/auth/Auth.API/API/Program.cs

```diff
@@ -19,16 +19,22 @@
 using OpenTelemetry.Metrics;
 using OpenTelemetry.Trace;
 using Serilog;
+using Serilog.Enrichers.Span;
 using Serilog.Formatting.Json;
 
-var logger = new LoggerConfiguration()
-    .WriteTo.Console(new JsonFormatter())
-    .CreateLogger();
-
 var builder = WebApplication.CreateBuilder(args);
 
+var log = new LoggerConfiguration()
+    .Filter.ByExcluding("RequestPath like '/health%'")
```

Comment:
Should this be healtz?

Reply:
This should match `healthz`, `healthx`, `health`, `healthiness` etc.